webpack/ForemanTasks/Components/TasksTable/TasksTablePage.js

```diff
@@ -40,16 +40,20 @@ const TasksTablePage = ({ getBreadcrumbs, history, ...props }) => {
     }));
   };
 
-  const TaskSelectedAction = (id, name) => {
-    props.actionSelected(id, name, url);
+  const TaskSelectedAction = {
+    [CANCEL]: () => {
+      props.bulkCancel(getSelected(), url);
```

Comment:
Could assign `getSelected()` to a `const` so you don't have to run it twice here.